data_processors/pipeline/lambdas/bcl_convert.py

```diff
@@ -307,22 +307,21 @@ def handler(event, context) -> dict:
             # Get the values in the override cycles column
             override_cycles_list = sample_group_df["override_cycles"].unique().tolist()
 
-            # Get the samplesheet midfix and also the output directory for each batch
-            if len(override_cycles_list) == 1:
-                batch_name = "{}_{}".format(sample_type, assay)
-            else:
-                batch_name = None
+        # Get the samplesheet midfix and also the output directory for each batch
+        if len(override_cycles_list) == 1:
```

Comment:
Did the indentation change here? 
Does that not affect the logic?

Reply:
Indentation looks ok to me; done checking in PyCharm.

Reply:
Ah; wait! Unit tests are failing. Looks like Indentation play up here.

Reply:
Fixed